source/module_base/test/tool_threading_test.cpp

```diff
@@ -8,6 +8,8 @@
 * - Tested functions of class threading tool:
 *   - TASK_DIST_1D:
 *       - (template)Distributing 1d tasks by worker id (int and long long)
+*   - BLOCK_TASK_DIST_1D:
+*       - (template)Distributing 1d tasks by block_size and worker id (int and long long)
 **/
 class tool_threading_test : public testing::Test

```

Comment:
This class seems useless here.

Reply:
> This class seems useless here.

Indeed, I didn't distinguish between TEST and TEST_F.